x/msgfees/spec/05_events.md

```diff
@@ -8,9 +8,10 @@ Existing fee event continue to show total fee charged
 
 ## Any Tx
 
-| Type     | Attribute Key | Attribute Value    |
-| -------- | ------------- | ------------------ |
-| message  | fee           | total fee (coins)  |
+| Type     | Attribute Key    | Attribute Value               |
+| -------- |------------------|-------------------------------|
+| message  | fee              | total fee (coins)             |
+| message  | min_fee_charged  | floor gas price * gas (coins) |
 
 If additional fee is assessed, these events will also be emitted (reason for not always emitting them mainly saving space on block output)
 
```

Comment:
I saw you comment in my other PR.  We could probably merge some of those changes to this then.  I added a bit more details to the Attribute Value sections.  I also moved the `basefee` attribute out of the `## Tx with Additional Fee` and into the `## Any Tx` section.  This is because `basefee` is emitted even if there isn't an msg with and additional fee.

Reply:
The `baseFee` is only emitted when a tx is successful, but both `fee` and `min_fee_charged` can be emitted even if the `tx` fails (if the floor fee was charged). Also, now, the `baseFee` isn't emitted when there aren't any msg-based fees (since it's exactly the same as `fee` in that case).

I'll update this stuff to be clearer I guess.

Reply:
on the section below for `## Tx with Additional Fee` probably should have `fee`, `basefee`, `min_fee_changed` ,`Additional Fee` also

Reply:
Docs updated.